extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -21,13 +21,19 @@ import {LocalSubscriptionPlatform} from './local-subscription-platform';
 import {PageConfig, PageConfigResolver} from '../../../third_party/subscriptions-project/config';
 import {Renderer} from './renderer';
 import {SubscriptionPlatform} from './subscription-platform';
-import {dev, user} from '../../../src/log';
 import {installStylesForDoc} from '../../../src/style-installer';
 import {tryParseJson} from '../../../src/json';
+import {user} from '../../../src/log';
 
 /** @const */
 const TAG = 'amp-subscriptions';
 
+/** @typedef {{serviceId: string, authorizationUrl: string}} */
+export let ServiceDefination;
```

Comment:
This probably not going to work very well, because the configs between platforms will differ significantly. E.g. `authorizationUrl` is only local thing.

Reply:
Done